packages/constellation/fileTransformer.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line
+const path = require('path')
+
+module.exports = {
+  process(src, filename) {
+    return `module.exports = ${JSON.stringify(path.basename(filename))};`
```

Comment:
what's the file all about?

Reply:
I had to go beyond stubbing out the svg files for Jest/Storyshots to correctly resolve/consume icons and components using icons.

This resolved errors like the following when running the whole suite of snapshot tests:
<img width="1240" alt="Screen Shot 2022-08-02 at 2 39 57 PM" src="https://user-images.githubusercontent.com/12600902/182496885-d28ae588-8c4f-4dd9-ad93-c15d7dfa054d.png">

Reply:
@blockdylanb let me know if that makes sense. Also happy to leave a comment in this file to describe why it is needed.

Reply:
@noahub i see, if you could leave a comment that would be great, thanks!